delete-project-next-item/index.js

```diff
@@ -0,0 +1,113 @@
+const core = require("@actions/core");
+const { graphql } = require("@octokit/graphql");
+
+const token =  core.getInput('github-token');
+const organization = core.getInput('organization');
+const projectNumber = core.getInput('project-number');
+const resourceNodeId = core.getInput('resource-node-id');
+
+const graphqlWithAuth = graphql.defaults({
+  headers: {
+    authorization: `token ${token}`,
+  },
+});
+
+async function getProjectId() {
+  console.log(`Getting project with number ${projectNumber}`);
+
+  try {
+    const { organization: {projectNext: { id }}} = await graphqlWithAuth(`
+    query ($org: String!, $number: Int!) {
+      organization(login: $org) {
+        projectNext(number: $number) {
+          id
+          fields(first:20) {
+            nodes {
+              id
+              name
+              settings
+            }
+          }
+        }
+      }
+    }`,
+    {
+      org: organization,
+      number: projectNumber
+    });
+    console.log(`Got project id ${id}`);
+    return id;
+  } catch (error) {
+    core.setFailed(error.message);
+    console.log(error.message);
+  }
+}
+
+async function getItemId(projectId) {
+  console.log(`Getting item id from node id ${resourceNodeId}`);
+
+  try {
+    const { addProjectNextItem: { projectNextItem: { id }}} = await graphqlWithAuth(`
+    mutation(
+      $project:ID!,
+      $resource_id:ID!
+    ) {
+      addProjectNextItem(
```

Comment:
Think you want to do something else here? 🤔

Reply:
I tried to find a better way to get just the item ID, but had no luck. `addProjectNextItem` is idempotent, and the item should already be on the project board, so this would just return the item ID